modules/tftp/server.rb

```diff
@@ -108,6 +108,28 @@ def pxeconfig_file(mac)
     end
   end
 
+  class Pxegrub2targetos < Server
+    def setup_bootloader(mac, os)
+      FileUtils.mkdir_p(pxeconfig_dir(mac))
+      bootloader_directory = (Proxy::TFTP::Plugin.settings.bootloader_universe || "/usr/local/share/bootloader-universe")
```

Comment:
Or should it just error out if no bootloader is set?

Reply:
I now raise an error, in case the directory is not configured in the settings or one of the binary files is not present in the OS-dependent directory.